Cannot build: babel-minify gives syntax error

I am unable to build the library using the makefile. I get the following error on my computer.

```
mkdir -p build
cd src; cat Database.js Component.js Glyph.js ImageSpread.js Pcoord.js PcoordCanvas.js PcoordSVG.js Query.js ScatterPlot.js ScatterPlotCanvas.js ScatterPlotSVG.js | babel-minify > ../build/CinemaComponents.v2.4.1.min.js
/usr/local/lib/node_modules/babel-minify/lib/cli.js:39
function printHelpInfo({
                       ^

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/babel-minify/bin/minify.js:3:1)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
makefile:6: recipe for target 'minify' failed
make: *** [minify] Error 1
```

Ideally, we should find a better solution for minify-ing the code as having to install nodejs, npm and babel-minify just to build the project is pretty absurd. Perhaps it would be best to forego minify-ing altogether and simply concatenate the source files together because at least that won't require any dependencies.

--- a/makefile
+++ b/makefile
@@ -1,11 +1,16 @@
 FILES=Database.js Component.js Glyph.js ImageSpread.js Pcoord.js PcoordCanvas.js PcoordSVG.js Query.js ScatterPlot.js ScatterPlotCanvas.js ScatterPlotSVG.js
 VERSION=$(shell cat version)
-OUTPUT_PREFIX=CinemaComponents.v$(VERSION).min
+OUTPUT_PREFIX=CinemaComponents.v$(VERSION)
 
-minify:
+full:
 	mkdir -p build
-	cd src; cat $(FILES) | babel-minify > ../build/$(OUTPUT_PREFIX).js
-	cat css/*.css > build/$(OUTPUT_PREFIX).css
+	cd src; cat $(FILES) > ../build/$(OUTPUT_PREFIX).js
+	cp build/$(OUTPUT_PREFIX).js build/$(OUTPUT_PREFIX).min.js
+	cat css/*.css > build/$(OUTPUT_PREFIX).min.css
+
+minify: full
+	mkdir -p build
+	cat build/$(OUTPUT_PREFIX).js | babel-minify > build/$(OUTPUT_PREFIX).min.js
 
 build/cinemascience.github.io:
 	cd build; git clone https://github.com/cinemascience/cinemascience.github.io.git

